src/.vuepress/markdown/code-snippet.js

```diff
@@ -47,11 +47,27 @@ module.exports = function snippet(md, options = {}) {
       rawPath = rawPath.replace(snippetId, '');
     }
 
+    let highlights = rawPath.match(/{.*}/);
+    if (highlights && highlights[0]) {
+      highlights = highlights[0];
+      rawPath = rawPath.replace(highlights, '');
+    }
+
+    let language = rawPath.match(/\[([a-z]*)\]/);
+    if (language && language[0]) {
+      rawPath = rawPath.replace(language[0], '');
+      if (language[1]) {
```

Comment:
We could use ternary ?
```      language = language[1] ? language[1] : null; ```

Reply:
Simpler: `language = language[1] || null`